src/intermediate.rs

```diff
@@ -11,6 +11,23 @@ use crate::utils::{
     is_identifier_user_defined,
 };
 
+#[derive(Debug)]
```

Comment:
I had originally added an extra field and for that made it its own struct but later deleted that field. I still think it's worth it to keep the change anyway as it's more readable vs a tuple.